src/applications/edu-benefits/tests/components/SchoolSelectField.unit.spec.jsx

```diff
@@ -172,4 +172,26 @@ describe('<SchoolSelectField>', () => {
     expect(onChange.firstCall.args[0]).to.eql('test');
     expect(selectInstitution.firstCall.args[0]).to.eql(institutions[0]);
   });
+
+  it('should call setCannotFindSchool prop when manual entry link clicked', () => {
+    const setCannotFindSchool = sinon.spy();
+    const tree = mount(<SchoolSelectField
```

Comment:
One of the nice side effects of a flat prop structure / dumb views is that they're very easy to test. They don't do anything but display information and call props.